plugins/LadspaEffect/caps/CMakeLists.txt

```diff
@@ -28,11 +28,6 @@ IF(LMMS_BUILD_WIN32)
 	)
 ENDIF(LMMS_BUILD_WIN32)
 
-IF(NOT LMMS_BUILD_APPLE AND NOT LMMS_BUILD_OPENBSD)
-	SET_TARGET_PROPERTIES(caps PROPERTIES LINK_FLAGS "${LINK_FLAGS} -shared -Wl,-no-undefined")
-ENDIF(NOT LMMS_BUILD_APPLE AND NOT LMMS_BUILD_OPENBSD)
-
-IF(LMMS_BUILD_LINUX)
-	SET_TARGET_PROPERTIES(caps PROPERTIES LINK_FLAGS "${LINK_FLAGS}")
-ENDIF(LMMS_BUILD_LINUX)
-
+IF(NOT LMMS_BUILD_APPLE AND NOT LMMS_BUILD_OPENBSD AND NOT MSVC)
+	target_link_libraries(caps PUBLIC "-shared -Wl -no-undefined")
```

Comment:
I believe this line is causing the build fail. Is it fine to remove this block. 

I looked at git blame and saw that this is a tobydox commit so might not be necessary.

Reply:
I removed the block. I'll see if CI passes.

Reply:
Not really fixed but resolving to have one less review.